jumpscale/packages/stellar_stats/bottle/stellar_stats.py

```diff
@@ -25,6 +25,12 @@ def get_stats():
     tokencode = query_params.get("tokencode", "TFT")
     detailed = j.data.serializers.json.loads(query_params.get("detailed", "false"))
     res = {}
+
+    # cache the request in local redis
+    redis = j.clients.redis.get("redis_instance")
+    if redis.get(f"{network}-{tokencode}-{detailed}"):
```

Comment:
can do 

```python
cached_data = redis.get(f"{network}-{tokencode}-{detailed}")
if cached_data:
   return cached_data
```
instead of going there twice